packages/cli/src/controllers/me.controller.ts

```diff
@@ -270,13 +257,11 @@ export class MeController {
 	 * Update the logged-in user's settings.
 	 */
 	@Patch('/settings')
-	async updateCurrentUserSettings(req: MeRequest.UserSettingsUpdate): Promise<User['settings']> {
-		const payload = plainToInstance(UserSettingsUpdatePayload, req.body, {
-			excludeExtraneousValues: true,
-		});
-
-		await validateEntity(payload);
-
+	async updateCurrentUserSettings(
+		req: AuthenticatedRequest,
+		_: Response,
+		@Body payload: SettingsUpdateRequestDto,
```

Comment:
In future it'd be nice to allow to only pass in the args that are actually used, e.g. here only the payload.

Reply:
That's how I was originally doing this in the [previous attempt](https://github.com/n8n-io/n8n/pull/9734). but I think someone suggested not to do that, so I decided to skip that change in this PR. 
If we can get a consensus on this, I can port over the rest of the changes from that PR.